repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/headerNormalization/HeaderNormalizationTest.groovy

```diff
@@ -242,4 +244,52 @@ class HeaderNormalizationTest extends ReposeValveTest {
         //"content-type" | "application/xMl"
         //"Content-Type" | "APPLICATION/xml"
     }
+
+    def "Should not split request headers when configured as such with merge-header filter"() {
```

Comment:
This test shouldn't be part of this test class. It should be in it's own test class with it's own configuration, not tacked onto a test for a different filter.